athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/serde/v2/ListTablesResponseSerDeTest.java

```diff
@@ -47,7 +47,7 @@ public void beforeTest()
     {
         expected = new ListTablesResponse("test-catalog", ImmutableList.of(
                 new TableName("schema1", "table1"),
-                new TableName("schema1", "table2")));
+                new TableName("schema1", "table2")), null);
```

Comment:
better to use real values?

Reply:
A null is a real value that can be passed in CP.